app/models/user.rb

```diff
@@ -3,6 +3,9 @@ class User < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :name, presence: true
+  validates :posts_counter, comparison: { greater_than_or_equal_to: 0 }, presence: true
+
```

Comment:
- [ ] The `comparison` validation helper doesn't ensure the value is numerical. You can use [`numericality`](https://guides.rubyonrails.org/active_record_validations.html#numericality) validation helper instead of it.


> PostsCounter must be an **integer** greater than zero.

Reply:
Done! 👍